rust_src/build.rs

```diff
@@ -110,8 +110,19 @@ where
             },
 
             ParseState::LispFn(name) => {
-                if line.starts_with("pub") || line.starts_with("fn") {
-                    if let Some(func) = name.or_else(|| get_function_name(&line)) {
+                let name = name.or_else(|| get_function_name(&line));
+
+                if line.starts_with("fn") {
+                    panic!(format!(
```

Comment:
You don't need `format!` in `panic!`, it supports format args out of the box.

Reply:
Which one is it that doesn't? I know one of the macros complains when I give it format args. I thought it was panic.

Reply:
I seem to remember something similar, but right now I don't find any. It might have been fixed some Rust versions ago.

Reply:
Good to know.